edalize/yosys.py

```diff
@@ -19,12 +19,37 @@ def get_doc(cls, api_ver):
                          'desc' : 'Output file format. Legal values are *json*, *edif*, *blif*'},
                         {'name' : 'yosys_as_subtool',
                          'type' : 'bool',
-                         'desc' : 'Determines if Yosys is run as a part of bigger toolchain, or as a standalone tool'}],
+                         'desc' : 'Determines if Yosys is run as a part of bigger toolchain, or as a standalone tool'},
+                        {'name' : 'makefile_name',
+                         'type' : 'String',
+                         'desc' : 'Generated makefile name, defaults to $name.mk'},
+
+                        {'name' : 'script_name',
+                         'type' : 'String',
+                         'desc' : 'Generated tcl script filename, defaults to $name.mk'},
+                        ],
                     'lists' : [
                         {'name' : 'yosys_synth_options',
                          'type' : 'String',
-                         'desc' : 'Additional options for the synth command'}
-                        , ]}
+                         'desc' : 'Additional options for the synth command'},
+                        ]}
+
+    @classmethod
+    def validate_args(cls, args):
```

Comment:
Do we need this function? I suspect it was added to handle argument parsing, but that should not be done by Edalize anymore

Reply:
Actually no, removed.